kubernetes/ama-logs.yaml

```diff
@@ -538,17 +538,16 @@ spec:
         nodeAffinity:
           requiredDuringSchedulingIgnoredDuringExecution:
             nodeSelectorTerms:
-              - labelSelector:
-                matchExpressions:
-      # kubernetes.io/os label doesnt exist in k8s versions < 1.14  so make sure to choose label based on k8s version in aks yaml
-                  - key: kubernetes.io/os
-                    operator: In
-                    values:
-                    - linux
-                  - key: type
-                    operator: NotIn
-                    values:
-                      - virtual-kubelet
+            - matchExpressions:
```

Comment:
as we discussed, will have to remove in the chart yaml. Please validate and removing this doesnt have any impact on the update of the extension/chart.